tools/validators/instance_validator/README.md

```diff
@@ -16,3 +16,19 @@ In case of you extended the ontology by adding new types, run the following: `py
 When using a modified ontology, ensure you follow the folder-naming convention: `digitalbuildings/ontology/yaml`. This will allow the instance validator to rely on the new types in the ontology.
 
 Note: as of the current development stage, you must clone the entire repository and run this instance validator script from this directory.
+
+### Telemetry validation
+
+The validator supports a telemetry validation mode. When this mode is enabled, the validator will listen on a provided pubsub subscription for telemetry messages, and validate the message contents against the instance configuration. It is recommended that you first use the instance validator with telemetry validation mode disabled, and then enable it after that passes.
+
+If you would like to use the telemetry validation mode, you must provide the "--subscription" and "--service-account" parameters when running instance_validator.py. Failure to provide both of these parameters will result in early termination of the validator and an error message. If you do not provide either parameter, the validator will run with telemetry validation mode disabled.
+
+The --subscription parameter value should be a fully-qualified path to a Google Cloud Pubsub subscription, e.g. projects/google.com:your-project/subscriptions/your-subscription.
+
+The --service-account parameter value should be a JSON file corresponding to a service account that has permission to pull messages from the subscription.
```

Comment:
can we have an example of this?  It's not intuitive how a service account (which I think if as looking like and email address or a file path) would map to a JSON file.

Reply:
just a path to the service-account-file